group_vars/abid/production.yml

```diff
@@ -12,8 +12,7 @@ passenger_app_env: "production"
 passenger_ruby: "/usr/bin/ruby2.6"
 ruby_version_override: "ruby2.6"
 bundler_version: "2.2.19"
-nodejs__upstream_release: 'node_12.x'
-nodejs__upstream_key_id: '68576280'
+desired_nodejs_version: "v12.22.6"
```

Comment:
This is not a blocking criticism, however I am curious as to why the minor and patch versions are not tracking the latest release for https://nodejs.org/download/release/latest-erbium/?

Reply:
The problem we encountered with latest is some folks preferred to pin versions and brought this up.